### ISSUE TITLE
Harmonize access to Socrates from PROTEUS

### ISSUE THREAD

**Issue by lsoucasse** [FormingWorlds/PROTEUS#134]
At the moment JANUS and AGNI have their own tools to download and install SOCRATES. When you use PROTEUS, you should be able to switch JANUS / AGNI while having only one SOCRATES code installed.

**Comment by nichollsh** [comment]
Definitely agree. AGNI only needs the RAD_DIR environment variable to be set.

**Comment by lsoucasse** [comment]
Just tested to swap from Janus to Agni using the same Socrates repository. It just works fine. When installing the Julia Agni package it just adds the julia wrapper into the socrates repo. I think we just need to remove the get_socrates command in the documentation when using AGNI, assuming socrates is already there as it is a core module.

**Comment by nichollsh** [comment]
That's great - glad to hear it works nicely. Presumably we will then add a get_socrates script of some form into PROTEUS?

**Comment by lsoucasse** [comment]
Yes, when we remove Socrates as a submodule we need to write a cli command that runs the download and the basic compiling of Socrates. Then if you install agni it will build the julia wrapper on top of it.

**Comment by lsoucasse** [comment]
Closing this issue, I will address the documentation update when working on #135.
